pkgsCross.aarch64-multiplatform.expect: configure: error: Expect can't be cross compiled

**Describe the bug**
expect cannot be cross compiled on master of as of writing.

**To Reproduce**
Steps to reproduce the behavior:
1. nix-build -A pkgsCross.aarch64-multiplatform.expect

**Additional context**

```
unpacking sources
unpacking source archive /nix/store/cq0s1r8b0ppzb6z7ny9v8pvsbslyy48h-expect5.45.4.tar.gz
source root is expect5.45.4
setting SOURCE_DATE_EPOCH to timestamp 1517752543 of file expect5.45.4/tclconfig/tcl.m4
patching sources
autoreconfPhase
autoreconf: export WARNINGS=
autoreconf: warning: autoconf input should be named 'configure.ac', not 'configure.in'
autoreconf: Entering directory '.'
autoreconf: configure.in: not using Gettext
autoreconf: configure.in: not using aclocal
autoreconf: configure.in: tracing
autoreconf: configure.in: adding subdirectory testsuite to autoreconf
autoreconf: Entering directory 'testsuite'
autoreconf: configure.in: not using Gettext
autoreconf: configure.in: not using aclocal
autoreconf: configure.in: tracing
autoreconf: configure.in: not using Libtool
autoreconf: configure.in: not using Intltool
autoreconf: configure.in: not using Gtkdoc
autoreconf: running: /nix/store/y1f3d3b7jjcsix58vyksglagdmf2jyw0-autoconf-2.70/bin/autoconf --force
configure.in:8: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:8: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
../tclconfig/tcl.m4:45: TEA_PATH_TCLCONFIG is expanded from...
configure.in:8: the top level
configure.in:11: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:11: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
../tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:11: the top level
configure.in:11: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:11: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
../tclconfig/tcl.m4:2159: TEA_MISSING_POSIX_HEADERS is expanded from...
../tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:11: the top level
configure.in:11: warning: The macro `AC_HAVE_HEADERS' is obsolete.
configure.in:11: You should run autoupdate.
./lib/autoconf/oldnames.m4:35: AC_HAVE_HEADERS is expanded from...
../tclconfig/tcl.m4:2159: TEA_MISSING_POSIX_HEADERS is expanded from...
../tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:11: the top level
configure.in:14: warning: AC_OUTPUT should be used without arguments.
configure.in:14: You should run autoupdate.
configure.in:14: warning: AC_C_BIGENDIAN should be used with AC_CONFIG_HEADERS
autoreconf: configure.in: not using Autoheader
autoreconf: configure.in: not using Automake
autoreconf: '../tclconfig/install-sh' is updated
autoreconf: Leaving directory 'testsuite'
autoreconf: configure.in: not using Libtool
autoreconf: configure.in: not using Intltool
autoreconf: configure.in: not using Gtkdoc
autoreconf: running: /nix/store/y1f3d3b7jjcsix58vyksglagdmf2jyw0-autoconf-2.70/bin/autoconf --force
configure.in:21: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:21: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
tclconfig/tcl.m4:45: TEA_PATH_TCLCONFIG is expanded from...
configure.in:21: the top level
configure.in:46: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:46: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:46: the top level
configure.in:46: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:46: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
tclconfig/tcl.m4:2159: TEA_MISSING_POSIX_HEADERS is expanded from...
tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:46: the top level
configure.in:46: warning: The macro `AC_HAVE_HEADERS' is obsolete.
configure.in:46: You should run autoupdate.
./lib/autoconf/oldnames.m4:35: AC_HAVE_HEADERS is expanded from...
tclconfig/tcl.m4:2159: TEA_MISSING_POSIX_HEADERS is expanded from...
tclconfig/tcl.m4:3121: TEA_SETUP_COMPILER is expanded from...
configure.in:46: the top level
configure.in:80: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:80: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
tclconfig/tcl.m4:738: TEA_ENABLE_THREADS is expanded from...
configure.in:80: the top level
configure.in:87: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:87: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
tclconfig/tcl.m4:680: TEA_ENABLE_SHARED is expanded from...
configure.in:87: the top level
configure.in:95: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
tclconfig/tcl.m4:2581: TEA_TCL_EARLY_FLAG is expanded from...
tclconfig/tcl.m4:2594: TEA_TCL_EARLY_FLAGS is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
tclconfig/tcl.m4:2629: TEA_TCL_64BIT_FLAGS is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:95: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:95: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
tclconfig/tcl.m4:2629: TEA_TCL_64BIT_FLAGS is expanded from...
tclconfig/tcl.m4:1051: TEA_CONFIG_CFLAGS is expanded from...
configure.in:95: the top level
configure.in:101: warning: The macro `AC_HELP_STRING' is obsolete.
configure.in:101: You should run autoupdate.
./lib/autoconf/general.m4:203: AC_HELP_STRING is expanded from...
tclconfig/tcl.m4:867: TEA_ENABLE_SYMBOLS is expanded from...
configure.in:101: the top level
configure.in:141: warning: The macro `AC_HAVE_HEADERS' is obsolete.
configure.in:141: You should run autoupdate.
./lib/autoconf/oldnames.m4:35: AC_HAVE_HEADERS is expanded from...
configure.in:141: the top level
configure.in:150: warning: The macro `AC_TYPE_SIGNAL' is obsolete.
configure.in:150: You should run autoupdate.
./lib/autoconf/types.m4:776: AC_TYPE_SIGNAL is expanded from...
configure.in:150: the top level
configure.in:156: warning: The macro `AC_HEADER_TIME' is obsolete.
configure.in:156: You should run autoupdate.
./lib/autoconf/headers.m4:743: AC_HEADER_TIME is expanded from...
tclconfig/tcl.m4:2374: TEA_TIME_HANDLER is expanded from...
configure.in:156: the top level
configure.in:156: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.in:156: You should run autoupdate.
./lib/autoconf/general.m4:2846: AC_TRY_COMPILE is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2248: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2269: AC_CACHE_CHECK is expanded from...
tclconfig/tcl.m4:2374: TEA_TIME_HANDLER is expanded from...
configure.in:156: the top level
configure.in:168: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:168: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
configure.in:168: the top level
configure.in:254: warning: The macro `AC_CANONICAL_SYSTEM' is obsolete.
configure.in:254: You should run autoupdate.
./lib/autoconf/general.m4:2080: AC_CANONICAL_SYSTEM is expanded from...
configure.in:254: the top level
configure.in:455: warning: The macro `AC_TRY_LINK' is obsolete.
configure.in:455: You should run autoupdate.
./lib/autoconf/general.m4:2919: AC_TRY_LINK is expanded from...
configure.in:455: the top level
configure.in:470: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:470: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:470: the top level
configure.in:489: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:489: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:489: the top level
configure.in:537: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:537: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:537: the top level
configure.in:642: warning: The macro `AC_HEADER_EGREP' is obsolete.
configure.in:642: You should run autoupdate.
./lib/autoconf/oldnames.m4:37: AC_HEADER_EGREP is expanded from...
configure.in:642: the top level
configure.in:715: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:715: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:715: the top level
configure.in:740: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:740: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:740: the top level
configure.in:757: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:757: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:757: the top level
configure.in:779: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:779: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:779: the top level
configure.in:801: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:801: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:801: the top level
configure.in:825: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:825: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:825: the top level
configure.in:868: warning: The macro `AC_HAVE_FUNCS' is obsolete.
configure.in:868: You should run autoupdate.
./lib/autoconf/functions.m4:230: AC_HAVE_FUNCS is expanded from...
configure.in:868: the top level
configure.in:869: warning: The macro `AC_HAVE_FUNCS' is obsolete.
configure.in:869: You should run autoupdate.
./lib/autoconf/functions.m4:230: AC_HAVE_FUNCS is expanded from...
configure.in:869: the top level
configure.in:878: warning: The macro `AC_TRY_RUN' is obsolete.
configure.in:878: You should run autoupdate.
./lib/autoconf/general.m4:2996: AC_TRY_RUN is expanded from...
configure.in:878: the top level
configure.in:1059: warning: AC_OUTPUT should be used without arguments.
configure.in:1059: You should run autoupdate.
configure.in:1059: warning: AC_C_BIGENDIAN should be used with AC_CONFIG_HEADERS
autoreconf: configure.in: not using Autoheader
autoreconf: configure.in: not using Automake
autoreconf: 'tclconfig/config.sub' is updated
autoreconf: 'tclconfig/config.guess' is updated
autoreconf: Leaving directory '.'
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./tclconfig/config.sub
Updating Autotools / GNU config script to a newer upstream version: ./tclconfig/config.guess
configuring
configure flags: --prefix=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu --bindir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/bin --sbindir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/sbin --includedir=/nix/store/0ljc03kv4xscjq0wn58pdqpr1y73cvzz-expect-5.45.4-aarch64-unknown-linux-gnu-dev/include --oldincludedir=/nix/store/0ljc03kv4xscjq0wn58pdqpr1y73cvzz-expect-5.45.4-aarch64-unknown-linux-gnu-dev/include --mandir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/share/man --infodir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/share/info --docdir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/share/doc/expect --libdir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/lib --libexecdir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/libexec --localedir=/nix/store/3idp9cq9lcrdpwwd2dndnc8j9ahp3yzl-expect-5.45.4-aarch64-unknown-linux-gnu/share/locale --with-tcl=/nix/store/9xmaq96qwq63ghd01qazndmfl60nrpvy-tcl-8.6.11-aarch64-unknown-linux-gnu/lib --with-tclinclude=/nix/store/9xmaq96qwq63ghd01qazndmfl60nrpvy-tcl-8.6.11-aarch64-unknown-linux-gnu/include --exec-prefix=\$\{out\} --build=x86_64-unknown-linux-gnu --host=aarch64-unknown-linux-gnu
checking for correct TEA configuration... ok (TEA 3.9)
configure: configuring expect 5.45.4
checking for Tcl configuration... found /nix/store/9xmaq96qwq63ghd01qazndmfl60nrpvy-tcl-8.6.11-aarch64-unknown-linux-gnu/lib/tclConfig.sh
checking for existence of /nix/store/9xmaq96qwq63ghd01qazndmfl60nrpvy-tcl-8.6.11-aarch64-unknown-linux-gnu/lib/tclConfig.sh... loading
checking for a BSD-compatible install... /nix/store/w21pgi6691202iafl0cnr7hk6wvvdz3n-coreutils-8.32/bin/install -c
checking for aarch64-unknown-linux-gnu-gcc... aarch64-unknown-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-unknown-linux-gnu-gcc accepts -g... yes
checking for aarch64-unknown-linux-gnu-gcc option to enable C11 features... none needed
checking how to run the C preprocessor... aarch64-unknown-linux-gnu-gcc -E
checking whether make sets $(MAKE)... yes
checking for aarch64-unknown-linux-gnu-ranlib... aarch64-unknown-linux-gnu-ranlib
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for sys/time.h... yes
checking for grep that handles long lines and -e... /nix/store/3mx947xcmsyk79izqc8ifv31qafp80pc-gnugrep-3.6/bin/grep
checking for egrep... /nix/store/3mx947xcmsyk79izqc8ifv31qafp80pc-gnugrep-3.6/bin/grep -E
checking if the compiler understands -pipe... yes
checking whether byte ordering is bigendian... no
checking for sin... no
checking for main in -lieee... no
checking for main in -linet... no
checking for net/errno.h... no
checking for connect... yes
checking for gethostbyname... yes
checking dirent.h... yes
checking for errno.h... yes
checking for float.h... yes
checking for values.h... yes
checking for limits.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for sys/wait.h... yes
checking for dlfcn.h... yes
checking for sys/param.h... yes
checking for Tcl public headers... /nix/store/9xmaq96qwq63ghd01qazndmfl60nrpvy-tcl-8.6.11-aarch64-unknown-linux-gnu/include
checking for Tcl private include files... private headers found with public headers
checking for pthread_mutex_init in -lpthread... yes
checking for building with threads... yes (default)
checking how to build libraries... shared
checking if 64bit support is requested... no
checking if 64bit Sparc VIS support is requested... no
checking if compiler supports visibility "hidden"... yes
checking if rpath support is requested... yes
checking system version... Linux-5.4.78-2-pve
checking for ar... aarch64-unknown-linux-gnu-ar
checking for required early compiler flags...  _LARGEFILE64_SOURCE
checking for 64-bit integer type... using long
checking for build with symbols... no
checking for sys/select.h... yes
checking for sys/sysmacros.h... yes
checking for varargs.h... no
checking for sys/stropts.h... no
checking for sys/sysconfig.h... no
checking for sys/fcntl.h... yes
checking for sys/ptem.h... no
checking for sys/strredir.h... no
checking for sys/strpty.h... no
checking for sys/bsdtypes.h... no
checking return type of signal handlers... void
checking whether struct tm is in sys/time.h or time.h... time.h
checking how aarch64-unknown-linux-gnu-gcc reports undeclared, standard C functions... error
checking for sys/time.h... (cached) yes
checking for struct tm.tm_zone... yes
checking for gmtime_r... yes
checking for localtime_r... yes
checking tm_tzadj in struct tm... no
checking tm_gmtoff in struct tm... yes
checking long timezone variable... yes
checking union wait... no
checking for gethostname... yes
checking for socket... yes
checking for select... yes
checking for getpseudotty... no
checking for getpseudotty in -lseq... no
checking for openpty... no
checking for openpty in -lutil... yes
checking if running Sequent running SVR4... no
checking build system type... x86_64-unknown-linux-gnu
checking host system type... aarch64-unknown-linux-gnu
checking target system type... aarch64-unknown-linux-gnu
checking for pid_t... yes
checking if running Mach... no
checking if running MachTen... no
checking if on Pyramid... no
checking if on Apollo... no
checking if on Interactive... no
checking stty to use... /nix/store/w21pgi6691202iafl0cnr7hk6wvvdz3n-coreutils-8.32/bin/stty
checking if stty reads stdout... no
checking if running Solaris... no
checking for memmove... yes
checking for sysconf... yes
checking for strftime... yes
checking for strchr... yes
checking for timezone... yes
checking for siglongjmp... yes
checking for memcpy... yes
checking if WNOHANG requires _POSIX_SOURCE... configure: error: Expect can't be cross compiled
builder for '/nix/store/0438hlrn6jhgcygzc26i0jlab7c93c0d-expect-5.45.4-aarch64-unknown-linux-gnu.drv' failed with exit code 1
```

@siraben Do you have any idea?

--- a/pkgs/tools/misc/expect/default.nix
+++ b/pkgs/tools/misc/expect/default.nix
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     (fetchpatch {
-      url = "https://raw.githubusercontent.com/buildroot/buildroot/master/package/expect/0001-enable-cross-compilation.patch";
+      url = "https://raw.githubusercontent.com/buildroot/buildroot/c05e6aa361a4049eabd8b21eb64a34899ef83fc7/package/expect/0001-enable-cross-compilation.patch";
       sha256 = "1jwx2l1slidvcpahxbyqs942l81jd62rzbxliyd9lwysk38c8b6b";
     })
   ];

